ext/json/ext/parser/parser.rl

```diff
@@ -173,6 +200,100 @@ static VALUE rsymbol_cache_fetch(rvalue_cache *cache, const char *str, const lon
     return rsymbol;
 }
 
+/* rvalue stack */
+
+#define RVALUE_STACK_INITIAL_CAPA 128
```

Comment:
I can squeeze some more perf by bumping the initial capacity, but the concern is that it might overflow the stack.

`activitypub.json` goes to `93` deep., `ctim_catalog.json` to `390`, and `twitter.json` to `239`.

For most users it would be fine to allocate this much on the stack given most modern system have a 8MB stack, but I fear people using `alpine/musl` might run into trouble given the default stack size there is only `128KB`.

If we were to ROFLscale this to 512 entries, that would use 4KiB on the stack. In addition with `512B` from the initial `fbuffer` and ~120B for the `JSON_Parser` struct, that may be a bit much.

Reply:
Actually, the difference is really minimal, so I don't think it's really worth it.